python/py-hid/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-hid
 version             1.0.5
-revision            0
+revision            1
```

Comment:
no need to increase the "revision" for this situation. The py311 port will be build and for the others will just become unavailable.

Reply:
fixed